zerver/tests/test_messages.py

```diff
@@ -956,6 +956,16 @@ def test_reaction(self) -> None:
         self.assertEqual(msg_dict['reactions'][0]['user']['full_name'],
                          sender.full_name)
 
+    def test_missing_anchor(self) -> None:
+        self.login(self.example_email("hamlet"))
+        result = self.client_get('/json/messages?'
+                                 'use_first_unread_anchor=false&'
+                                 'num_before=1&'
+                                 'num_after=1')
```

Comment:
This wrapping is super hard to read.  I'd prefer you either construct a query dict and use `urllib` to convert it, or just wrap like this:
```
        result = self.client_get(
            '/json/messages?use_first_unread_anchor=false&...
￼
```